python-threatexchange/threatexchange/extensions/manifest.py

```diff
@@ -48,4 +48,35 @@ def load_from_module_name(
 
         if not isinstance(manifest, cls):
             raise ValueError(f"TX_MANIFEST is not a {cls.__name__}!")
+
+        try:
+            manifest.bootstrap()
+        except Exception as exc:
+            raise ValueError(f'Manifest failed to bootstrap: {module_name}') from exc
+
+        try:
+            manifest.verify()
+        except Exception as exc:
+            raise ValueError(f'Manifest failed verification: {module_name}') from exc
+
```

Comment:
blocking: Per previous comment these shouldn't be added here.

Instead, in the CLI, call these two functions in the extensions command

Reply:
oop - apologies missed this the first time. Just updated